File: views/src/main/java/dlgdev/weighttracker/dagger/WeightCheckerActivityComponent.java
package dlgdev.weighttracker.dagger;

import dagger.Component;
import dlgdev.weighttracker.views.checkers.WeightTrackerActivity;

@Component(modules = {WeightCheckerActivityModule.class, NavigationControllerModule.class,
		DateFormatModule.class, RepositoriesModule.class})
public interface WeightCheckerActivityComponent {
	void inject(WeightTrackerActivity activity);
}

File: views/src/main/java/dlgdev/weighttracker/dagger/WeightCheckerActivityComponent.kt
package dlgdev.weighttracker.dagger

import dagger.Component
import dlgdev.weighttracker.views.checkers.WeightTrackerActivity

@Component(modules = arrayOf(WeightCheckerActivityModule::class, NavigationControllerModule::class, DateFormatModule::class, RepositoriesModule::class))
interface WeightCheckerActivityComponent {
    fun inject(activity: WeightTrackerActivity)
}
